_layouts/default.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en"><head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.7.1 -->
+  <title>QIoT project | QIoT: Quarkus and Internet of Things</title>
+  <meta name="generator" content="Jekyll v4.1.1" />
+  <meta property="og:title" content="QIoT project" />
+  <meta property="og:locale" content="en_US" />
+  <meta name="description" content="QIoT: Quarkus and Internet of Things" />
+  <meta property="og:description" content="QIoT: Quarkus and Internet of Things" />
+  <link rel="canonical" href="http://localhost:4000/" />
+  <meta property="og:url" content="http://localhost:4000/" />
+  <meta property="og:site_name" content="QIoT project" />
+  <meta name="twitter:card" content="summary" />
+  <meta property="twitter:title" content="QIoT project" />
+  <script type="application/ld+json">
+  {"description":"QIoT: Quarkus and Internet of Things","@type":"WebSite","headline":"QIoT project","url":"http://localhost:4000/","name":"QIoT project","@context":"https://schema.org"}</script>
+  <!-- End Jekyll SEO tag -->
+  <link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="QIoT project" /></head>
```

Comment:
Can be the feed.xml relative path?

Reply:
No idea !

Reply:
It seems, https://qiot-project.github.io/feed.xml does the trick!